test/background.isolation.test.ts

```diff
@@ -359,6 +359,83 @@ preferencesTestSet.map(preferences => {
                   });
                 });
               });
+
+              describe('toggle isolation off', () => {
+                beforeEach(async () => {
+                  bg.tmp.storage.local.preferences.isolation.global.navigation.action =
+                    'always';
+                  bg.tmp.storage.local.preferences.isolation.active = true; // default to true
+                  bg.tmp.isolation.setActiveState(false); // toggle off
```

Comment:
Most of the tests use "integration / blackbox testing" where directly calling internal functions is avoided. So in this case also yielding the command would be good, since it allows the internal API to change without having to adjust the tests.

Reply:
I'm very new to this testing suite...  I'm not sure what you're suggesting here.  (sorry)

Reply:
No worries, what I meant is what you did in the other tests already,

```js
          browser.commands.onCommand.addListener.yield('toggle_isolation');
```

instead of directly calling `setActiveState`

Reply:
Got it.  Resolved.  (And clarified a couple of the descriptions to describe from the browser pov.)